Assets/UGSSamples/FriendsSample/Scripts/Common/Managers/RelationshipsManager.cs

```diff
@@ -88,8 +87,8 @@ void UIInit()
         async Task LogInAsync()
         {
             var playerID = AuthenticationService.Instance.PlayerId;
-            m_SamplePlayerProfileService = new SamplePlayerProfileService();
-            m_LoggedPlayerProfile = new PlayerProfile(m_SamplePlayerProfileService.GetName(playerID), playerID);
+            var playerName = await AuthenticationService.Instance.GetPlayerNameAsync();

```

Comment:
Do we actually want to keep the abstraction layer we had in place (with the interface and the SamplePlayerProfileImplementation)? The AuthenticationService.Instance.GetPlayerNameAsync() is particular to UGS, but nothing prevents developers to inherit from IPlayerProfile and to create their custom way to generate the name. However, we could assume that we implement it the easiest way like this and then expect them to override if needed.  @UnityJacob what do you think?